.github/workflows/build_and_test.yml

```diff
@@ -173,17 +173,17 @@ jobs:
       - name: TurboRepo local server
         uses: felixmosh/turborepo-gh-artifacts@v1
         with:
-          repo-token: ${{ secrets.RC_TURBO_GH_TOKEN }}
-          server-token: ${{ secrets.TURBO_SERVER_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Ideally, we should just add a new secret from the github repo setting instead of replacing with common. Maybe it has less permission/access.

Reply:
NOTE: ear will create a personal access token and include as github secret.  Permissions == read only.  Use secret names above. @bhardwajaditya will revert this change.

Reply:
@bhardwajaditya I created the access token and configured the secrets, but the build is still failing.  It looks like it's failing on the "Post TurboRepo local server" step, which appears to be something from the github action used here:  

- name: TurboRepo local server
        uses: felixmosh/turborepo-gh-artifacts@v1

So it almost seems like it's not our issue per se?  Not sure.  Can you look into this please and figure out what that 'felixmosh/turborepo-gh-artifacts' action is trying to do in it's post step?  Thanks.

Reply:
@ear-dev the error says repo token is not configured

Reply:
@bhardwajaditya Well, I did configure the secrets, but perhaps re-running the workflow only re-runs in the last state. Can you trigger a new run from your branch to get a clean run/test?

Reply:
I have no idea why it is not finding the secrets.RC_TURBO_GH_TOKEN when I have set it on both the repo and the organizational level.  I'm tempted to just use the GITHUB_TOKEN.

Reply:
@ear-dev I am also trying to configure and run the same on my branch once.

Reply:
@ear-dev it worked for me after adding these tokens on my branch

Reply:
@bhardwajaditya where did you create the secret?  Are they actions secrets at the repository level?  I also configured these but am still getting this error.  Any ideas why? I'm not sure what's going on...

![image](https://user-images.githubusercontent.com/28864717/211338725-196d00f3-0bd9-4a3c-833e-3f0a196e7df8.png)

Reply:
@bhardwajaditya at this point I am tempted to go back to using GITHUB_TOKEN, unless we can find a quick fix for this.  Thanks.

Reply:
@ear-dev probably it's not working in this PR since it's on my fork. We can merge and it should work then

Reply:
Ok, that's a good point.... I'll give it a try.